app/build.gradle

```diff
@@ -90,7 +90,9 @@ dependencies {
 
 
     //Finix Payment Library
-    implementation files('libs/app-release.aar')
+//    implementation files('/Users/tariq.ebadi/AndroidStudioProjects/finix-android-payment-sheet/finix-android-payment-sheet/build/outputs/aar/finix-android-payment-sheet-release.aar')
```

Comment:
you probably don't want to expose these local file names within the public app

Reply:
updated with comments removed